### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BalintMendli**

### CODE REVIEW

**Review by zlayabekrija — COMMENTED** [review]
Hello, @zenott first try of this project is great, now let's focus on some important things here.
We should set up the root page, so when we hit localhost:3000 (example), it should show us either login page or some static page (up to your choice). 
Next, our attendance model is not set up to add any users for our events, please let's make this working.
Please let's make these changes any resubmit review request.
Happy coding.

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class EventsController < ApplicationController
+  def new
```
Comment (dipto0321) [inline app/controllers/events_controller.rb:4]:
```ruby
 if current_user
      @event = current_user.events.build
    else
      redirect_to events_path
    end
```
This is okay to do some authentication, but rails have some filters to do this thing more proper way. You can play with that



Here you can get [more](https://guides.rubyonrails.org/action_controller_overview.html#filters)

**Review by dipto0321 — APPROVED** [review]
HI @zenott ,

All is good now. Just a small thing I commented below. So, you already fix all those problems and in 2nd review, I didn't find any noticeable issue. For this, I can say you are good to move forward. This is from me 👍 .

Thanks 🙏

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (BalintMendli) [reply app/controllers/events_controller.rb:4]: Fixed. Thanks for the review!

**Pull request merged**